policy-engine/src/graph.rs

```diff
@@ -45,37 +45,65 @@ pub(crate) async fn index(
     V1_GRAPH_INCOMING_REQS.inc();
 
     // Check that the client can accept JSON media type.
-    commons::ensure_content_type(req.headers(), CONTENT_TYPE)?;
+    commons::ensure_content_type(req.headers(), CONTENT_TYPE)
+        .map_err(|e| api_response_error(&req, e))?;
 
     // Check for required client parameters.
     let mandatory_params = &app_data.mandatory_params;
-    commons::ensure_query_params(mandatory_params, req.query_string())?;
+    commons::ensure_query_params(mandatory_params, req.query_string())
+        .map_err(|e| api_response_error(&req, e))?;
```

Comment:
Instead of wrapping each possible error generator, should we just make an internal `_index(...)` (or some such, I dunno what Rust's conventions are for naming private functions) and then have:

```rust
pub(crate) async fn index(
    req: HttpRequest,
    app_data: actix_web::web::Data<AppState>,
) -> Result<HttpResponse, GraphError> {
    _index(req, app_data).map_err(|e| api_response_error(&reg, e))?
}
```

?  Otherwise, I'm worried that there will be errors we forget to wrap.  I guess #185 is the furthest up we can bubble things, and I'm agnostic on whether we want to do something at the `App::new().wrap(...)` level or the `index` level, but I'm pretty sure we don't want to have multiple `map_err` within one function, unless only a subset of the functions errors deserve the `api_response_error` mapping.